config/routes.rb

```diff
@@ -21,6 +20,12 @@
   get 'people', to: 'relationships#index'
   resources :relationships, only: [:destroy, :create]
 
+  get 'forgot_password', to: "forgot_passwords#new"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.